packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -159,13 +162,13 @@ export function setValueForProperty(node, name, value) {
     if (shouldIgnoreValue(propertyInfo, value)) {
       deleteValueForProperty(node, name);
       return;
-    } else if (propertyInfo.mustUseProperty) {
+    } else if (shouldUseProperty(name)) {
       // Contrary to `setAttribute`, object properties are properly
       // `toString`ed by IE8/9.
-      node[propertyInfo.propertyName] = value;
+      node[name] = value;
     } else {
-      var attributeName = propertyInfo.attributeName;
-      var namespace = propertyInfo.attributeNamespace;
+      var attributeName = getAttributeName(name);
+      var namespace = getAttributeNamespace(name);
       // `setAttribute` with objects becomes only `[object]` in IE8/9,
       // ('' + value) makes it output the correct toString()-value.
```

Comment:
Is this true?

<img width="1104" alt="screen shot 2017-11-30 at 8 02 51 pm" src="https://user-images.githubusercontent.com/590904/33463092-756c98b2-d609-11e7-8871-b6d1e1765f92.png">

Reply:
Also valueOf:

<img width="752" alt="screen shot 2017-11-30 at 8 03 38 pm" src="https://user-images.githubusercontent.com/590904/33463117-948460b8-d609-11e7-975a-ba7274d527fa.png">

Reply:
Maybe only IE8?

Reply:
(Regardless, don't want to change the logic here, only the structure :-)

Reply:
Totally. It's just nice to look at this code again with a fresh set of eyes. I'll file an issue.

Reply:
https://github.com/facebook/react/issues/11735